packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin2.tsx

```diff
@@ -481,6 +496,10 @@ const getStyles = (theme: GrafanaTheme2) => ({
     border: `1px solid ${theme.colors.border.weak}`,
     boxShadow: theme.shadows.z2,
     userSelect: 'text',
+    width: `${maxWidth}px`,
```

Comment:
let's change this to be a `maxWidth` property. i know this will make the tooltip width adjust to content (like the old one), but it makes the tooltip compact instead of taking up a minimum amount of space even when the content is small.

Reply:
it looks actually not that bad:
<img width="272" alt="Screenshot 2024-01-17 at 16 25 39" src="https://github.com/grafana/grafana/assets/1418924/e97361d8-b570-41e2-92be-e4f894e70951">
<img width="312" alt="Screenshot 2024-01-17 at 16 34 43" src="https://github.com/grafana/grafana/assets/1418924/01a64e0d-8c8b-4bd9-823a-a98e6602cdb4">
<img width="204" alt="Screenshot 2024-01-17 at 16 34 50" src="https://github.com/grafana/grafana/assets/1418924/379dd2ec-4fbb-4302-bfc0-242f99c22442">
<img width="309" alt="Screenshot 2024-01-17 at 16 34 59" src="https://github.com/grafana/grafana/assets/1418924/533db622-9e6a-4f78-83ab-8fbdeba439a3">
<img width="308" alt="Screenshot 2024-01-17 at 16 35 04" src="https://github.com/grafana/grafana/assets/1418924/4b0d2a22-400f-46f3-a5c0-92b919107b15">

Reply:
<img width="272" alt="Screenshot 2024-01-17 at 16 25 39" src="https://github.com/grafana/grafana/assets/1418924/e97361d8-b570-41e2-92be-e4f894e70951">

maybe some minimum right margin for labels here?